app/presenters/tree_builder_miq_action_category.rb

```diff
@@ -0,0 +1,53 @@
+class TreeBuilderMiqActionCategory < TreeBuilder
+  has_kids_for Classification, [:x_get_tree_classification_kids]
+
+  private
+
+  def override(node, object, _pid, _options)
+    img = "blank.gif"
+    leaf = !object.entries.any?
+    img = "tag.png" unless leaf
+    node[:image] = ActionController::Base.helpers.image_path("100/#{img}")
+    node[:title] = object.description
```

Comment:
Setting `node[:title]` here seems completely superfluous, it already is that way.